aptos-move/e2e-move-tests/src/tests/existence_check.rs

```diff
@@ -0,0 +1,88 @@
+// Copyright © Aptos Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{assert_success, tests::common, MoveHarness};
+use aptos_package_builder::PackageBuilder;
+use aptos_types::{
+    account_address::AccountAddress, move_utils::MemberId, on_chain_config::FeatureFlag,
+};
+use std::{path::Path, str::FromStr};
+
+fn run_transactions(features: Vec<FeatureFlag>, path: &Path, function_names: &[&str]) -> u64 {
+    let mut h = MoveHarness::new_with_features(features, vec![]);
+    let acc = h.new_account_at(AccountAddress::from_hex_literal("0x123").unwrap());
+    assert_success!(h.publish_package(&acc, path));
+
+    h.run_entry_function(
+        &acc,
+        MemberId::from_str("0x123::test::init").unwrap(),
+        vec![],
+        vec![],
+    );
+    let mut txns = vec![];
+    for name in function_names.iter() {
+        txns.push(h.create_entry_function(
+            &acc,
+            MemberId::from_str(format!("0x123::test::{name}").as_str()).unwrap(),
+            vec![],
+            vec![],
+        ));
+    }
+    let statuses = h.run_block_get_output(txns);
+    statuses.iter().map(|out| out.gas_used()).sum()
+}
+
+fn check_for_transactions(path: &Path, function_names: &[&str]) {
+    assert_eq!(
+        run_transactions(vec![], path, function_names),
```

Comment:
So for the comment above: I think you need to pass enabled features vector and disabled features vector